front/README.md

```diff
@@ -39,36 +39,16 @@ Launches end to end tests.
 
 It requires:
 
-- Backend containers to be up: `docker compose build core redis editoast postgres`
-- Running front
-- [`poetry`](https://python-poetry.org/docs/#installation) to be installed
```

Comment:
Is poetry no longer needed?

Reply:
It is not indeed. The entire test suite has been changed for frontend tests. Note that you'll still need `poetry` for the global e2e test suite (in `osrd/tests`).